scanner/node.go

```diff
@@ -1,18 +1,90 @@
 package scanner
 
+import (
+	"encoding/json"
+	"os"
+	"os/exec"
+	"strings"
+)
+
 func configureNode(sourceDir string) (*SourceInfo, error) {
 	if !checksPass(sourceDir, fileExists("package.json")) {
 		return nil, nil
 	}
 
+	data, err := os.ReadFile("package.json")
+	if err != nil {
+		return nil, nil
+	}
+
+	var result map[string]interface{}
+	err = json.Unmarshal(data, &result)
+	if err != nil {
+		return nil, nil
+	}
+
 	s := &SourceInfo{
-		Builder: "heroku/buildpacks:20",
-		Family:  "NodeJS",
-		Port:    8080,
+		Family: "NodeJS",
+		Port:   8080,
 		Env: map[string]string{
 			"PORT": "8080",
 		},
 	}
 
+	scripts, ok := result["scripts"].(map[string]interface{})
+
+	if !ok || scripts["start"] == nil {
```

Comment:
Probably doesn't need to be this version but we could prompt for the script or output info about why we did buildpacks instead?

Reply:
I'm hesitant to rewrite the package.json.  What's more likely to happen is people are going to identify other scripts that should be used.  For example, I've seen package.json files with client and server scripts - we would be interested in the latter.

Reply:
Good point, weird to have this update code in their realm. Perhaps a note that we've configured the build to run `run build` is probably enough - maybe enough to point them in a direction if they choose to update it themselves.